test/fast/Unit tests/nvm_install_binary_nosource

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+cleanup () {
+  nvm cache clear
+  nvm deactivate
+  rm -rf ${NVM_DIR}/v*
+  nvm unalias default
+}
+
+die () { echo "$@" ; cleanup; exit 1;}
+
+
+\. ../../../nvm.sh
+
+
+nvm_binary_available() {
+    return 1
+}
+
+# Unit test to check if the function errors out when the flag is set
+OUTPUT="$(nvm_install_binary node std 8.0.0 1 2>&1)"
+EXPECTED_OUTPUT='Binary download failed.Download from source aborted'
+if [[ "${OUTPUT}" != *"${EXPECTED_OUTPUT}"* ]]; then
```

Comment:
these are bash-only comparisons which i'd expect to fail in sh/dash/ksh.

i'll update them to use posix comparisons.

Reply:
Sure